ThreeJs4Net/Geometries/EdgesGeometry.cs

```diff
@@ -1,6 +1,90 @@
-﻿namespace ThreeJs4Net.Geometries
+﻿using System.Collections.Generic;
+using ThreeJs4Net.Core;
+using ThreeJs4Net.Math;
+
+
+namespace ThreeJs4Net.Geometries
 {
-    public class EdgesGeometry
-    {
-    }
+	public class EdgesGeometry : BufferGeometry
+	{
+		/// <summary>
+		/// This can be used as a helper object to view the edges of a geometry.
+		/// </summary>
+		/// <param name="geometry"> Any geometry object. </param>
+		/// <param name="thresholdAngle">  An edge is only rendered if the angle (in degrees) between the face normals of the adjoining faces exceeds this value. default = 1 degree. </param>
+		public EdgesGeometry(BufferGeometry geometry, int thresholdAngle = 1)
+		{
+			this.type = "EdgesGeometry";
+
+			// buffer
+
+			List<float> vertices = new List<float>();
+
+			// helper variables
+
+			var thresholdDot = Mathf.Cos(MathUtils.DEG2RAD * thresholdAngle);
+			int[] edge = new int[2];
+			IDictionary<string, Dictionary<string, int>> edges = new Dictionary<string, Dictionary<string, int>>();
+			int edge1;
+			int edge2;
+			string key;
+			string[] keys = { "A", "B", "C" };
+
+			// prepare source geometry
+
+			var geometry2 = new Geometry().FromBufferGeometry(geometry);
+
+			geometry2.MergeVertices();
+			geometry2.ComputeFaceNormals();
+
+			var sourceVertices = geometry2.Vertices;
+			var faces = geometry2.Faces;
+
+			// now create a data structure where each entry represents an edge with its adjoining faces
+
+			for (int i = 0; i < faces.Count; i++)
+			{
+				var face = faces[i];
+
+				for (int j = 0; j < 3; j++)
+				{
+					edge1 = (int)face.GetType().GetField(keys[j]).GetValue(face);
+					edge2 = (int)face.GetType().GetField(keys[(j + 1) % 3]).GetValue(face);
+					edge[0] = Mathf.Min(edge1, edge2);
+					edge[1] = Mathf.Max(edge1, edge2);
+
+					key = edge[0].ToString() + ',' + edge[1].ToString();
```

Comment:
Not wrong, but this is the pattern we should be using:
string key = $"{edge[0]},{edge[1]}";